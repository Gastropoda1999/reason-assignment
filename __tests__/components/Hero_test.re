open Jest;
open Expect;

let _ =

describe "Hero" (fun _ => {
  test "renders" (fun _ => {
    let tree = Hero.createElement children::[] () |> Renderer.render;

    (expect tree) |> toMatchSnapshot;
  });
});
