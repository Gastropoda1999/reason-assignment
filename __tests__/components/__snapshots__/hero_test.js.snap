// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero renders 1`] = `
<div
  className={
    Object {
      "data-css-m5x6pd": "",
    }
  }
>
  <Title
    reasonProps={0}
  />
  <Features
    reasonProps={0}
  />
</div>
`;
